### ISSUE TITLE
Model exporting issues.

### ISSUE THREAD

**Issue by 09williamsad** [Turk645/Xenoblade-Switch-Model-Importer-Noesis#1]
A number of models will get stuck when exporting to fbx or crash noesis when exporting to dae.
chr\ch\ch01011013.wismt being an example.

GLTF exports, but the exported model will freeze blender 2.79 and 2.93 on import.
[ch01011013.zip](https://github.com/Turk645/Xenoblade-Switch-Model-Importer-Noesis/files/9375466/ch01011013.zip)
On attempting to convert the gltf to fbx in noesis, noesis crashes.

**Comment by Turk645** [comment]
Yeah. i noticed this after posting as well. I think it is something in my skeleton code and im looking into it. If you use the skeleton override correctly that model extracts no problem though.

**Comment by nickmux** [comment]
Yeah i noticed this too with noah's model from xenoblade chronicles 3

**Comment by Lord018** [comment]
Dumb question but what is the skeleton override?

Edit: Figured it out, Tools>Display plugin tools>Xenoblade Switch
